website/project/model.py

```diff
@@ -216,19 +236,52 @@ def get_content(self, auth):
         return self.content
 
     @classmethod
-    def find_unread(cls, user, node):
-        default_timestamp = datetime.datetime(1970, 1, 1, 12, 0, 0)
+    def find_n_unread(cls, user, node, page=None, root_id=None, check=False):
         n_unread = 0
```

Comment:
This variable is now unnecessary; it is never reassigned, so you can just `return 0` at the end of the method.